trustymail/domain.py

```diff
@@ -90,6 +116,10 @@ def generate_results(self):
 
                         "MX Record": self.has_mail(),
                         "Mail Servers": self.format_list(self.mail_servers),
+                        "Mail Server Ports Tested": self.format_list(list(map(lambda x:str(x), self.ports_tested))),
+                        "Mail Server Is Listening": self.format_list(list(map(lambda x:x + ";" + str(self.starttls_results[x]["is_listening"]), self.starttls_results.keys()))),
+                        "Mail Server Supports SMTP": self.format_list(list(map(lambda x:x + ";" + str(self.starttls_results[x]["supports_smtp"]), self.starttls_results.keys()))),
```

Comment:
Thoughts on only includes those hostnames/ports that return as `True`?

Reply:
Do you mean something like this?
```json
"Mail Server Supports SMTP": "blah.foo.com:467,mail.blah.foo.com:467"
```
I'm fine with that.

Reply:
@h-m-f-t, did you decide that this change is something you want?  If so I can make the change.